packages/api/README.md

```diff
@@ -1,5 +1,10 @@
-<div style="text-align:center;"><h1>@sapphire/plugin-api</h1>
-<h3>Plugin for <a href="https://github.com/sapphire-project/framework">@sapphire/framework</a> to expose a REST API</h3></div>
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire.png)
+
+# @sapphire/plugin-api
+
+Plugin for <a href="https://github.com/sapphire-project/framework">@sapphire/framework</a> to expose a REST API</h4>
```

Comment:
```suggestion
Plugin for <a href="https://github.com/sapphire-project/framework">@sapphire/framework</a> to expose a REST API.</h4>
```

and goddammit am I missing something because you're missing so many tags